packages/contentstack-audit/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
can we use prettier config instead of editor ?

Reply:
Sure, I can remove it and commit. However, I need not push one for prettier we have it in our CLI global config itself.

Reply:
that works